src/chrome/content/rules/boardreader.com.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	Redirect to HTTP:
+		support.boardreader.com
+
+	Time out:
+		clients.boardreader.com
+		dev01.boardreader.com
+		forum.boardreader.com
+		spider5.boardreader.com
+-->
+<ruleset name="boardreader.com">
+	<target host="boardreader.com" />
+	<target host="www.boardreader.com" />
+	<target host="api.boardreader.com" />
+	<target host="apidemo.boardreader.com" />
```

Comment:
`devtools.boardreader.com`

Reply:
I didn't notice the cert chain issue, sorry

Reply:
Remove it again?

Reply:
I'd rather move it to the comments

Reply:
Added to comments.

Reply:
Thanks!